src/main/java/seedu/address/model/person/LastContact.java

```diff
@@ -49,17 +49,15 @@ public static boolean isValidDateTime(String dateTime) {
             return true;
         }
         String trimmedDateTime = dateTime.trim();
-        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-uuuu HHmm")
-                .withResolverStyle(ResolverStyle.STRICT);
         try {
-            LocalDateTime.parse(trimmedDateTime, formatter);
+            LocalDateTime.parse(trimmedDateTime, DATETIME_FORMATTER);
             return true; // Successfully parsed, input matches the format
         } catch (DateTimeParseException e) {
             return false; // Parsing failed, input does not match the format
         }
     }
 
-    public boolean isLastContacted() {
```

Comment:
@Colex2000 same here, i updated the function name to hasLastContacted to match the change in field name. changed all places that uses this method as well

Reply:
Can, thanks!